config/webpack.dev.js

```diff
@@ -69,6 +69,22 @@ if (devMode === 'local') {
   plugins.push(
     new webpack.NormalModuleReplacementPlugin(/client_factory.ts/, 'client_factory.mock.ts')
   );
+  plugins.push(
+    new webpack.NormalModuleReplacementPlugin(/plan\/duck\/operations\.ts/, res => {
+      if (res.rawRequest === '../operations') return;
+      res.request = res.request.replace(/operation.\.ts/, 'mock/operations.mock.ts');
+      res.resource = res.resource.replace(/operation.\.ts/, 'mock/operations.mock.ts');
+      res.resourceResolveData.path = res.resourceResolveData.path.replace(/operation.\.ts/, 'mock/operations.mock.ts');
+    })
+  );
+  plugins.push(
+    new webpack.NormalModuleReplacementPlugin(/(?:cluster|storage|plan)\/duck\/sagas\.ts/, res => {
```

Comment:
This is something we'd like to avoid.

Would like to keep the mock interface at the client level.